packages/ag-charts-website/src/content/docs/key-features/_examples/enterprise-features-example/main.ts

```diff
@@ -42,8 +43,16 @@ const options: AgCartesianChartOptions = {
             type: 'line',
             xKey: 'year',
             yKey: 'spending',
+            marker: {
+                enabled: false,
+            },
         },
     ],
 };
 
-AgCharts.create(options);
+let chart = AgCharts.create(options);
+
+function reloadChart() {
+    chart.destroy();
+    chart = AgCharts.create(options);
+}
```

Comment:
The example generator can't handle this correctly for FWs (see e2e test failures).